packages/shared/lib/models/Sync.ts

```diff
@@ -134,6 +134,9 @@ export interface SyncDeploymentResult {
     version: string;
     providerConfigKey: string;
     type: SyncConfigType;
+    last_deployed?: Date;
+    models?: string | string[];
+    id?: number | undefined;
```

Comment:
Seems like `models` is always defined 
```suggestion
    last_deployed?: Date;
    models: string | string[];
    id?: number | undefined;
```

Also why would id and last_deployed be not defined?